src/components/atoms/Button.tsx

```diff
@@ -1,13 +1,13 @@
 import React, { FC } from 'react';
-import { Button as MatButton, IconButton as MatIconButton } from '@material-ui/core';
+import { Button as MatButton, ButtonProps, IconButton as MatIconButton } from '@material-ui/core';
 
-interface IProps {
+interface IProps extends ButtonProps {
   onClick?: () => any;
 }
 
 // todo: make standard buttons for entire app
-const StandardButton: FC<IProps> = ({ onClick, children }) => (
-  <MatButton variant="contained" color="primary" onClick={onClick}>
+const StandardButton: FC<IProps> = ({ type, onClick, children }) => (
+  <MatButton type={type} variant="contained" color="primary" onClick={onClick}>
```

Comment:
We prefer specific buttons over generic ones.
see my post about it
https://medium.com/@yuval.bar.levi/react-spread-attributes-dont-overuse-them-3f322a0ace22

Reply:
replace with :
```
const buttonType = {}
if(isSubmit) {
buttonType.type = 'submit';
}
<MatButton {...buttonType} variant="contained" color="primary" onClick={onClick}>
```